EnglishTranslation-master/SKILL.tsv

```diff
@@ -5912,22 +5912,22 @@ SKILL_20151224_005911	{#993399}{ol}[Magic] - [Holy]{/}{/}
 SKILL_20151224_005912	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Creates a magic circle that drastically increases your maximum HP for a period of time.
 SKILL_20151224_005913	Duration: #{CaptionTime}# seconds{nl}Magic Circle Duration: #{CaptionRatio2}# seconds{nl}Maximum HP: +#{CaptionRatio}#%
 SKILL_20151224_005914	{#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Summons the wagon of the goddess to attack enemies.
-SKILL_20151224_005915	Divide the name of an enemy into numbers and calculate a new number.
-SKILL_20151224_005916	Divide the name of an enemy and calculate a new number from the first and the last letter.
+SKILL_20151224_005915	Divide the numbers of an enemy's name and calculate a new number.
+SKILL_20151224_005916	Calculate a new number from the first and the last letter of the enemy's name.
 SKILL_20151224_005917	If the enemy is hit with the amount of the calculated number, the enemy is hit by the multiplication of the number.
 SKILL_20151224_005918	Final Attack Damage: #{CaptionRatio}#%
 SKILL_20151224_005919	Decrease the level of an enemy depending on the calculated number.
 SKILL_20151224_005920	Additional Enemy Level: -#{CaptionRatio}#
 SKILL_20151224_005921	Last Rites
-SKILL_20151224_005922	Makes the attacks of party members have the Holy property. The Holy property damage is increased in proportion to the caster's Sacrament level and becomes more powerful when the HP is under 40%.
+SKILL_20151224_005922	Makes the attacks of party members have the Holy property. The Holy property damage is increased in proportion to the caster's [Sacrament] level and becomes more powerful when HP is under 40%.
```

Comment:
`The Holy property damage is increased in proportion to the caster's [Sacrament] level and becomes more powerful when HP is under 40%.`

to

`Increased Holy property damage in proportion to the caster's [Sacrament] level which becomes more powerful when HP is below 40%.` ?